src/components/layout/footer.js

```diff
@@ -254,6 +255,40 @@ const mobile_accordion_header = {
     boxShadow: 'none',
 }
 
+export const FooterElement = ({footer_category, is_ppc_redirect } ) => {
+
+    return (
+        <>
+            {footer_category.hide_ppc && is_ppc_redirect === true ? <> </> :
+                <LinksCol>
+                <LinkWrapper>
+                    <Title>{footer_category.title}</Title>
+                </LinkWrapper>
+                {footer_category.links.map((link,idx) => (
+                    link.hide_ppc && is_ppc_redirect === true? <> </> :
+                    link.hide_eu ?
+                    <Show.NonEU key={idx}>
+                        <LinkWrapper>
+                            <Link {...link}>{link.text}</Link>
+                        </LinkWrapper>
+                    </Show.NonEU> :
+                    <LinkWrapper key={idx}>
+                        <Link {...link}>{link.text}</Link>
+                    </LinkWrapper>
```

Comment:
```suggestion
                    !link.hide_ppc && !is_ppc_redirect && (
                       link.hide_eu ?
                       <Show.NonEU key={idx}>
                            <LinkWrapper>
                               <Link {...link}>{link.text}</Link>
                             </LinkWrapper>
                       </Show.NonEU> :
                       <LinkWrapper key={idx}>
                           <Link {...link}>{link.text}</Link>
                       </LinkWrapper>
                    )
```
Not sure if this suggested code works but we should reverse the conditions to avoid sending a fragment